src/translations/sqlb_ar_SA.ts

```diff
@@ -546,41 +546,51 @@ If any of the other settings were altered for this database file you need to pro
         <source>Unix epoch to local time</source>
         <translation>عَصر لينكس إلى الوقت المحلي</translation>
     </message>
+    <message>
+        <location filename="../ColumnDisplayFormatDialog.cpp" line="32"/>
+        <source>WebKit / Chromium epoch to date</source>
+        <translation type="unfinished"></translation>
+    </message>
     <message>
         <location filename="../ColumnDisplayFormatDialog.cpp" line="33"/>
+        <source>WebKit / Chromium epoch to local time</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../ColumnDisplayFormatDialog.cpp" line="35"/>
         <source>Date as dd/mm/yyyy</source>
         <translation>التاريخ بتنسيق dd/mm/yyyy</translation>
     </message>
     <message>
-        <location filename="../ColumnDisplayFormatDialog.cpp" line="35"/>
```

Comment:
This PR seems to be changing the other language files as well?

At a guess, you've accidentally included the other ones after they were touched by lupdate or similar?

You should probably remove all of the changes to files that aren't actually needed, just so it's easier to review. :smile: